### PR TITLE
Update the FAQ page

### PR DESCRIPTION
**Pull request by aaronallen8455**

### CODE REVIEW

File: content/faq.md [hunk 1]
```diff
@@ -4,10 +4,132 @@ title: Frequently Asked Questions
...
+ -  Summer of Haskell cannot accept participants (contributors or mentors) from
+    Russia, Belarus, or the so-called Donetsk People's Republic ("DNR") and
+    Luhansk People's Republic ("LNR"), but there are no restrictions on
+    participants in Ukraine but outside of the DNR and LNR.
+
+---
+
+### How does the program work?
+Newcomers are mentored by experienced contributors, and learn how to work with
+open-source and improve their ability to write Haskell code. No previous open
```
Comment (david-christiansen) [inline content/faq.md]:
```suggestion
open source software development and improve their ability to write Haskell code. No previous open
```

File: content/faq.md [hunk 2]
```diff
@@ -4,10 +4,132 @@ title: Frequently Asked Questions
...
+source experience is needed to attend the program as a would-to-be contributor,
```
Comment (david-christiansen) [inline content/faq.md]:
```suggestion
source experience is needed to participate in the program as a contributor,
```

File: content/faq.md [hunk 3]
```diff
@@ -4,10 +4,132 @@ title: Frequently Asked Questions
...
+however you need to have some knowledge of writing Haskell code. The program is
+addressed to everyone; there's no need to be a student. All the contributors
+are paid for their work; even if the work is unfinished or the produced code
```
Comment (david-christiansen) [inline content/faq.md]: I think that we actually aren't "paying for work" per se - I believe it's legally important that this be a stipend, not a payment for work. Perhaps check how Google structures theirs, and also check in with Ryan?

File: content/faq.md [hunk 4]
```diff
@@ -4,10 +4,132 @@ title: Frequently Asked Questions
...
+### Is it possible to be a contributor and mentor at the same time?
+No, by definition, contributors are not experienced with open source while all
+the mentors are experienced contributors and maintainers.
+
+---
+
+### Why are you running this program?
+The Haskell ecosystem relies on volunteer work. The majority of tools and
+libraries are open source and we want to be aware and educate developers on the
+importance of OSS. Having active contributors and devoted maintainers is
```
Comment (david-christiansen) [inline content/faq.md]: Do our audience know the abbreviation "OSS"?

File: content/faq.md [hunk 5]
```diff
@@ -4,10 +4,132 @@ title: Frequently Asked Questions
...
+
+---
+
+### Can I take part in the project as a group of contributors?
+No, we accept only individuals as contributors.
+
+---
+
+### Can I submit more than one proposal?
+We accept two proposals from each candidate maximum.
```
Comment (david-christiansen) [inline content/faq.md]:
```suggestion
We accept at most two proposals from each candidate.
```

**Review by jaspervdj — APPROVED** [review]
LGTM, agree with @david-christiansen's suggestions.

**Pull request closed without merging**